mysql-test/suite/rpl/r/percona_bug_ps8106.result

```diff
@@ -5,6 +5,8 @@ Note	####	Storing MySQL user name or password information in the master info rep
 [connection master]
 SET @replica_parallel_workers_save = @@global.replica_parallel_workers;
 SET GLOBAL replica_parallel_workers = 0;
```

Comment:
May be set `replica_parallel_workers` to `1` in the `.test`file instead. It looks like `0` and `1` have the same meaning.`1` just doesn't trigger a warning.
https://dev.mysql.com/doc/refman/8.0/en/replication-options-replica.html#sysvar_replica_parallel_workers
```
Beginning with MySQL 8.0.30, setting this variable to 0 is deprecated, and doing so raises a warning; 0 as a permitted value for replica_parallel_workers is subject to removal in a future MySQL release; set it to 1 instead, which has the same effect.
```

Reply:
Done